protos/local/workflow_commands.proto

```diff
@@ -89,6 +89,9 @@ message ScheduleLocalActivity {
     /// a timer before retrying. It should pass the attempt number from a `DoBackoff` activity
     /// resolution.
     uint32 attempt = 4;
+
+    // TODO: Original seq num for sched-to-close timeouts?
```

Comment:
It might be necessary for timing out properly when the timer based backoff is involved. The local activity seq numbers are different after the timer backoff. They have to be associated somehow and only lang can really know that.

I'm adding the tests for that now, but I think this is probably the only way to do it.

Reply:
We could also just not allow any timeouts to exceed the timer threshold and make it clear in the docs that timeouts are per "local attempt", and after the backoff timer they are reset. That saves a lot of complexity here for a pretty rare use case (which it doesn't look like the other SDKs handle anyway)

Reply:
The other option would be to attach "original schedule time" rather than original seq number, which I think saves on complexity but still requires lang to pass it around like the attempt #

Reply:
I see, yeah, just ask lang to provide whatever you need.